jenkins-pipelines/longevity-double-cluster-with-schema-changes-with-raft.jenkinsfile

```diff
@@ -0,0 +1,13 @@
+#!groovy
+
+// trick from https://github.com/jenkinsci/workflow-cps-global-lib-plugin/pull/43
+def lib = library identifier: 'sct@snapshot', retriever: legacySCM(scm)
+
+longevityPipeline(
+    backend: 'aws',
+    region: 'eu-west-1',
+    test_name: 'longevity_test.LongevityTest.test_custom_time',
+    test_config: '''["test-cases/longevity/longevity-change-cluster-size-by-2-times.yaml", "configurations/raft/enable_raft_experimental.yaml"]''',
```

Comment:
Don't we need also to change now the "raft" to official flag or in addition to it?

Reply:
The new flag `consistent_cluster_management: true` will enable on 5.2 the functionality which was enabled by `experimenta_feature: raft`. The new flag will be enabled by default for new clusters according to https://github.com/scylladb/scylladb/issues/12470. 
the experimental flag `experimenta_feature: raft` will be used for next versions (master, 5.3) for other raft features

New flag consistent_cluster_management is already in scylla_yaml.py. I can switch default value for this parameter to True in new commit in this PR

Reply:
ok, so I'm not sure if for these longevities we need the raft experimental flag or not.

Reply:
For 5.2 it may be not need, but for master and for next version it could be used.